jupyter-docker/Dockerfile

```diff
@@ -105,12 +105,10 @@ RUN apt-get install -yq --no-install-recommends \
     python-tk \
     python-numpy \
     libz-dev \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/* \
 
  # NOTE! not sure why, but this must run before pip installation
  && useradd -m -s /bin/bash -N -u $UID $USER \
- # jessie's default pip doesn't work well with jupyter
+ # jessie's default pip doesn't work well with jupyter # what about stretch?
```

Comment:
yeah maybe this comment is superfluous

Reply:
in any case, it' confusing for the comment to refer to jessie when we're not on jessie